pylint/checkers/exceptions.py

```diff
@@ -302,22 +302,22 @@ def _check_misplaced_bare_raise(self, node):
         if not current or not isinstance(current.parent, expected):
             self.add_message("misplaced-bare-raise", node=node)
 
-    def _check_bad_exception_context(self, node: nodes.Raise) -> None:
-        """Verify that the exception context is properly set.
+    def _check_bad_exception_cause(self, node: nodes.Raise) -> None:
+        """Verify that the exception cause is properly set.
 
-        An exception context can be only `None` or an exception.
+        An exception cause can be only `None` or an exception.
         """
         cause = utils.safe_infer(node.cause)
         if cause in (astroid.Uninferable, None):
             return
 
         if isinstance(cause, nodes.Const):
             if cause.value is not None:
-                self.add_message("bad-exception-context", node=node)
+                self.add_message("bad-exception-cause", node=node)
```

Comment:
```suggestion
                self.add_message("bad-exception-cause", node=node, confidence=INFERENCE)
```
Unrelated to the current MR, but as we upgrade the whole functional output we might as well add the right confidence at the same time. This is going to require a new import ``from pylint.interfaces import INFERENCE`` that I can't suggest.